portfolio/src/main/webapp/style.css

```diff
@@ -143,6 +143,37 @@ body {
   margin: 1.5% 2.5% 1.5% 1%;
 }
 
+#travel-map{
```

Comment:
Nit: It's best to use classes (like `.travel-map`) instead of ids when writing page styles. ids are over-specific and can't be re-used.

On the other hand, ids are useful for:
1. Targeting an individual element with JavaScript (e.g. to initialize a Google Maps instance)
2. Labeling section headers or other interesting parts of the page (these can be linked to directly, and browsers will automatically scroll to the correct position when opening the page)